core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -704,6 +710,18 @@ public static long defaultServerIdleTimeoutMillis() {
         return DEFAULT_SERVER_IDLE_TIMEOUT_MILLIS;
     }
 
+    /**
+     * Returns the default option to whether prevent server staying in idle state when an HTTP/2 PING frame or
+     * OPTIONS * request is received or not. Note that this flag has no effect if server uses HTTP/1 connection.
```

Comment:
From the point of view of a server, `OPTIONS *` request is not a ping but just a request:
https://github.com/line/armeria/blob/6d50c358c99865b463a0f772f04d3e13a8bc50e1/core/src/main/java/com/linecorp/armeria/server/Http1RequestDecoder.java#L131
So how about just mentioning HTTP/2 PING?
```suggestion
     * Returns the default option that is preventing the server from staying in an idle state when
     * an HTTP/2 PING frame is received.
```
If this makes sense, please fix other Javadocs as well. 😉